docs/api/renderers.rst

```diff
@@ -3,71 +3,69 @@
 MyST Renderers
 --------------
 
-MyST-Parser builds on the mistletoe
-:ref:`core renderers <mistletoe:renderers/core>`
-by including the extended tokens, listed in :ref:`api/tokens`,
-and adding bridges to docutils/sphinx:
 
-HTML
-....
+These renderers take the markdown-it parsed token stream and convert it to
+the docutils AST. The sphinx renderer is a subclass of the docutils one,
+with some additional methods only available *via* sphinx
+.e.g. multi-document cross-referencing.
 
-.. autoclass:: myst_parser.html_renderer.HTMLRenderer
-    :special-members: __init__, __enter__, __exit__
-    :members: default_block_tokens, default_span_tokens
-    :undoc-members:
-    :member-order: alphabetical
-    :show-inheritance:
 
+Docutils
```

Comment:
from a dogfooding perspective I feel like these should be written in markdown :-)  (not blocking this PR though)

Reply:
The reason why its not is because, autodoc has to read the python docstrings as rST. I'm sure you could get it to read them as markdown (and rewrite all you doctrings as such), but I doubt its worth the hassle!